graylog2-server/src/main/java/org/graylog2/plugin/inputs/MessageInput.java

```diff
@@ -92,7 +92,7 @@ public abstract class MessageInput implements Stoppable {
     protected String persistId;
     protected DateTime createdAt;
     protected Boolean global = false;
-    protected IOState.Type desiredState = IOState.Type.RUNNING;
+    protected IOState.Type desiredState = IOState.Type.SETUP;
```

Comment:
I think in the development phase we should leave it at `RUNNING`. Otherwise we will create an input in setup mode even if the feature flag is off.

Reply:
I don't think it makes a difference - the correct starting state based on the feature flag is set in the `InputLauncher`.

Reply:
What I meant was that although the feature flag is off we create inputs whose `desired_state` is `SETUP`. I wanted to avoid that because it is confusing if the `desired_state` is `SETUP` but the input state is `RUNNING`. If we also want to change or rename the `SETUP` state we could not do this without a migration because it could already be in the database of the other developers.

Reply:
Good point. I already made the change.